README.md

```diff
@@ -1,4 +1,5 @@
 # Hearty Capsules
 
 ### Personal Project
-Supplements web app
\ No newline at end of file
+
+Supplements web app
```

Comment:
Making note here of the Node version you're developing with is important for (hypothetical) other contributors but also future you.

I always recommend https://asdf-vm.com/ for version management, which would mean making a `.tool-versions` file with a `nodejs` line listed.